README.md

```diff
@@ -1,77 +1,60 @@
 <!--
 
-This source file is part of the StanfordBDHG Template Application project
+This source file is part of the Spatial Continuity project
 
-SPDX-FileCopyrightText: 2023 Stanford University
+SPDX-FileCopyrightText: 2024 Paul Heidekrüger
 
 SPDX-License-Identifier: MIT
 
 -->
 
-# Template Application
+# Spatial Continuity
 
-[![Build and Test](https://github.com/StanfordBDHG/TemplateApplication/actions/workflows/build-and-test.yml/badge.svg)](https://github.com/StanfordBDHG/TemplateApplication/actions/workflows/build-and-test.yml)
-[![codecov](https://codecov.io/gh/StanfordBDHG/TemplateApplication/graph/badge.svg?token=ezY7o5Trsk)](https://codecov.io/gh/StanfordBDHG/TemplateApplication)
-[![DOI](https://zenodo.org/badge/587923964.svg)](https://zenodo.org/badge/latestdoi/587923964)
 
-This repository contains the StanfordBDHG Template Application. It serves as a template repository for projects requiring a mobile application using continuous integration and continuous delivery setup.
+## Overview
 
-## Continous Delivery Workflows
+This repository contains a implementation of _Spatial Continuity_.
+_Spatial Continuity_ is a novel concept that proposes a combination of spatial computing and large language models (LLM) to provide seamless transitions between virtual and real-world contexts for making the world more accessible to users with low vision.
+Ultimately, this should lead to a truly continuous experience where the friction that accessible technologies introduce, e.g., by setup and positioning, is fully removed. 
 
-### Beta Deployment
+This implementation is part of our work on the "Spatial Continuity: Investigating Use Cases of Spatial Computing for Users with Low Vision" poster, which was accepted at the [27th International ACM SIGACCESS Conference on Computers and Accessibility](https://assets25.sigaccess.org/).
 
-The Beta Deployment workflow is triggered when a new commit is added to the main branch. 
 
-It first runs the Build and Test workflow to ensure all tests are passing.
-Once the Build and Test workflow passes, it builds the iOS application so it can be archived and sent to [TestFlight](https://developer.apple.com/testflight/) for internal beta deployment.
+## Functionality
```

Comment:
It would be great to add some visuals here, either from the visionOS simulator or from the paper. 
Would be nice to have something visual here embedded in the README.